pkg/crc/adminhelper/hosts.go

```diff
@@ -22,13 +24,28 @@ func UpdateHostsFile(instanceIP string, hostnames ...string) error {
 }
 
 func AddToHostsFile(instanceIP string, hostnames ...string) error {
+	if err := validateHostnames(hostnames); err != nil {
+		return err
+	}
```

Comment:
This has been added to the admin helper code itself, do you want it here as well?

Reply:
yes I dropped the commit